CONTRIBUTING.rst

```diff
@@ -598,7 +598,7 @@ Styling problems must be resolved before the pull request can be accepted.
 
 All of these commands are available when you create the development environment.
 
-When you run the test locally, the tests in ``chemistry.py`` & ``biology.py`` are automatically skipped if you don't have the optional dependencies (e.g. ``rdkit``) installed.
+When you run the test locally, the tests in ``chemistry.py``, ``biology.py``, ``janitor.spark`` are automatically skipped if you don't have the optional dependencies (e.g. ``rdkit``) installed.
```

Comment:
Maybe say ``janitor/spark`` or ``spark`` directory following the style of ``xx.py``. Maybe @ericmjl  feels more strongly

Reply:
For consistency, yes, let's go with what @hectormz has suggested. 

```suggestion
When you run the test locally, the tests in ``chemistry.py``, ``biology.py``, and ``spark.py`` are automatically skipped if you don't have the optional dependencies (e.g. ``rdkit``) installed.
```